qp_klp/tests/test_metagenomic_step.py

```diff
@@ -48,14 +47,12 @@ def test_metagenomic_creation(self):
         self.assertIsNotNone(step)
 
     def test_metagenomic_convert_bcl_to_fastq(self):
-        self._delete_test_output()
```

Comment:
Out of curiosity, why aren't these needed?

Reply:
The base test class now contains a proper tearDown() method.

Reply:
see above.